### DISCUSSION TITLE
Welcome to servo Discussions!

### DISCUSSION THREAD

**Discussion by jdm** [servo/servo#27442] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with members of the Servo developer community, as well as developers interested in building projects that rely on Servo. We invite you to:
  * Ask questions about building, embedding, or contributing to Servo.
  * Discuss the current capabilities of the engine.
  * Share ideas for how to extend Servo in new ways.
  * Brainstorm or workshop ideas that aren't concrete enough for an issue.

  Remember that this is a community we build together, and all discussions must abide by the project [Code of conduct](https://github.com/servo/servo/blob/master/CONTRIBUTING.md#conduct).

  To get started, comment below with an introduction of yourself and tell us about your interest in Servo.

**Comment by goyalyashpal** [comment]
HI.
I am interested in testing Servo - as reading about Rust language feels exciting to me. I aint a developer though.\
For trying Servo, I have tried both Windows Zip and exe from www.servo.org , but it is showing me error - which i have reported at #27983. It seems like t'll require updating MS-VC++. Will do it some days later. Just thought to check in meanwhile (:

**Comment by CKalt** [comment]
Hi,
I'm new to Rust however not new to programming. Been programing C/C++ for many years. My current interest for systems level programming include Genetic Programming and creating a Prolog compiler and so I would like to try Rust out in these areas. Servo specifically seems great if indeed could evolve to complete with Firefox (is that too much dreaming?)

I live in Ann Arbor, Michigan.

Chris Kaltwasser
